src/items/external-blocks.md

```diff
@@ -131,7 +131,12 @@ specifies the kind of library with the following possible values:
 - `framework` — Indicates a macOS framework. This is only valid for macOS
   targets.
 
-The `name` key must be included if `kind` is specified.
+The `name` key must be included if `kind` is specified. Depending on the target, `name`
+is interpreted as follows:
+
+- `*-pc-windows-msvc` — Refers to the base name of the static library `name.lib`,
+or the base name of the import library `name.lib` accompanying a dynamic-link library `name.dll`.
+- All other targets — Refers to the base name of the shared library.
```

Comment:
I had a bit of a hard time following this. The term "base name" isn't defined, and doesn't match my normal understanding of "base name" when it comes to paths. It also seems a little vague (doesn't mention prefixes at all). 

Perhaps it could be worded something like this?

 The `name` is the library name without the platform's usual filename prefix
 and suffix.

> **Note:** For example, a link name of `foo` would search for a dynamic
> library named `libfoo.so` on Linux targets. On Windows MSVC targets it would
> search for the `foo.lib` import library.
>
> On Windows MSVC targets, to avoid ambiguity with staticlib filenames,
> `rustc` produces an import library named `foo.dll.lib` when generating a
> cdylib crate named `foo`. This means that if another crate wants to use that
> import library as-is, then the link name will need to be `foo.dll` to match
> that import library naming convention.

Reply:
Thanks for the suggestion. I can change "base name" to something else. 

However, I don't quite understand the 2nd part of your comment, are you suggesting to add both quoted paragraphs as well?

Reply:
Yea, instead of using the list, just provide a side-note that provides some hints about platform-specific concerns.